native-schema-locater/package.json

```diff
@@ -53,16 +53,16 @@
     "eslint": "^7.11.0"
   },
   "peerDependencies": {
-    "@itwin/ecschema-metadata": "3.0.0-dev.134",
-    "@itwin/ecschema-locaters": "3.0.0-dev.134"
+    "@itwin/ecschema-metadata": "3.0.2",
+    "@itwin/ecschema-locaters": "3.0.2"
   },
   "dependencies": {
-    "@itwin/core-bentley": "3.0.0-dev.134",
-    "@itwin/core-geometry": "3.0.0-dev.134",
-    "@bentley/imodelhub-client": "3.0.0-dev.127",
-    "@itwin/core-backend": "3.0.0-dev.134",
+    "@itwin/core-bentley": "3.0.2",
+    "@itwin/core-geometry": "3.0.2",
+    "@bentley/imodelhub-client": "3.0.0-dev.177",
+    "@itwin/core-backend": "3.0.2",
     "@bentley/backend-itwin-client": "3.0.0-dev.90",
-    "@itwin/core-common": "3.0.0-dev.134",
+    "@itwin/core-common": "3.0.2",
     "@bentley/itwin-client": "3.0.0-dev.134",
     "@bentley/rbac-client": "3.0.0-dev.5",
     "@bentley/telemetry-client": "3.0.0-dev.127",
```

Comment:
The client packages have been replaced with new packages here: https://github.com/iTwin/imodels-clients

This includes the imodelhub-client

Reply:
ok let me take a look

Reply:
Done

Reply:
Done